api/levity.proto

```diff
@@ -65,8 +65,8 @@ enum TaskStatusCode {
     // Task exited naturally, either of its own volition or after being killed.
     Finished = 2; 
 
-    // The did not respond to the kill signal and brutaly killed. Implies that 
-    // there There is exit code to return 
+    // The did not respond to the term signal and brutally killed. Implies that 
```

Comment:
Nit: "The did not respond" part reads a bit weird.

Reply:
Will fix in next MR